src/containers/trends/CBGSlicesAnimationContainer.js

```diff
@@ -141,13 +140,12 @@ export default class CBGSlicesAnimationContainer extends React.Component {
   render() {
     const { mungedData } = this.state;
     if (_.isEmpty(mungedData)) {
-      const { margins, svgDimensions } = this.props;
-      const xPos = (svgDimensions.width / 2) - margins.left + margins.right;
-      const yPos = (svgDimensions.height / 2) - margins.top + margins.bottom;
       return (
-        <text className={styles.noDataMsg} id="noDataMsg" x={xPos + 40} y={yPos}>
-          No CGM data for this time period :(
-        </text>
+        <NoData
+          message="No CGM data for this time period :("
```

Comment:
I'm quite surprised you didn't factor this out as a constant! Also, the message is the same for both except for the CGM vs. BGM difference, so perhaps let's just make a template string with a gap for CGM or BGM a defaultProp in NoData and pass in the data type? (And have `cbg` map to 'CGM' and `smbg` map to 'BGM' in some kind of set of constants...)